services-api/build.gradle

```diff
@@ -1,32 +1,7 @@
-apply plugin: 'info.solidsoft.pitest'
-
-jar {
-  enabled = true
-  archiveBaseName = 'civil-sdt-services-api'
-}
-
-dependencyCheck {
-    suppressionFile = "${rootDir}/config/owasp/suppressions.xml"
-}
-
 dependencies {
     implementation project(':domain')
     implementation project(':utils')
-    testImplementation project(":domain").sourceSets.unitTest.output
-    testImplementation project(":utils").sourceSets.unitTest.output
-
-    testImplementation project(":domain").sourceSets.integTest.output
-    testImplementation project(":utils").sourceSets.integTest.output
-}
-
-bootJar {
-    enabled = false
-}
-
-tasks.withType(Test) {
-    useJUnitPlatform()
-}
 
-sonarqube {
-  skipProject = true
-}
+    testImplementation project(path: ':domain', configuration: 'testOutput')
+    testImplementation project(path: ':utils', configuration: 'testOutput')
```

Comment:
Are these needed?  The services-api module doesn't have any tests so may not require anything from the domain and utils module tests.

Reply:
removed

Reply:
Confirmed